### DISCUSSION TITLE
installation error

### DISCUSSION THREAD

**Discussion by davistome** [Leantime/leantime#713] [General]
After doing all the installation and configuration procedures, configuration.php does not find mydomain/install it is giving 404.
Xampp + windows

**Comment by marcelfolaron** [comment]
Did you check if htaccess is working correctly? Can you verify that the domain is pointing to the "public/" folder within leantime?

**Comment by marcelfolaron** [comment]
What does your vhost config look like?
Usually something simple like htaccess not enabled or domain pointing to
the wrong folder
